hwdb.d/60-input-id.hwdb

```diff
@@ -75,3 +75,7 @@ id-input:modalias:input:b0003v046Dp4066e0111*
 id-input:modalias:input:b0003v068Ep00F2e0100*
  ID_INPUT_ACCELEROMETER=0
  ID_INPUT_JOYSTICK=1
+
+# TEX Shinobi Trackpoint
+id-input:modalias:input:b0003v04D9p0407*
+ ID_INPUT_POINTINGSTICK=1
```

Comment:
question: does the trackpoint have its own USB ID or does this hwdb snippet end up also tagging the keyboard as `ID_INPUT_POINTINGSTICK` now? (assuming that's on a separate device node)

Reply:
It's got four entries:

```
E: NAME="USB-HID Keyboard System Control"
E: MODALIAS=input:b0003v04D9p0407e0111-e0,1,4,k74,8E,8F,ram4,lsfw
E: NAME="USB-HID Keyboard Consumer Control"
E: MODALIAS=input:b0003v04D9p0407e0111-e0,1,4,k71,72,73,80,8C,90,9B,9C,9E,9F,A3,A4,A5,A6,AB,AC,AD,D9,ram4,lsfw
E: NAME="USB-HID Keyboard Mouse"
E: MODALIAS=input:b0003v04D9p0407e0111-e0,1,2,4,k110,111,112,113,114,115,116,117,r0,1,6,8,B,C,am4,lsfw
E: NAME="USB-HID Keyboard"
E: MODALIAS=input:b0003v04D9p0407e0111-e0,1,4,14,k71,72,73,74,75,77,79,7A,7B,7C,7D,7E,7F,80,81,82,83,84,85,86,87,88,89,8A,B3,B4,B7,B8,B9,BA,BB,BC,BD,BE,BF,C0,C1,C2,F0,ram4,lsfw
```

so it looks like I need to be more specific withe the modalias?  What do you recommend?

Reply:
Commit https://github.com/systemd/systemd/pull/29068/commits/53642a52b4a41e09ad67ddaa09bfa921d4ad87cd makes the modalias more specific to prevent tagging other aspects of the device.

Reply:
that works, the alternative would be to add custom lookup matching to `60-input-id.rules` so we can match on vid/pid + name. see `70-touchpad.rules` which can be copied and modified for the `input-id` prefix and the other match conditions. That would allow for
```
input-id:usb:v04D9p0407:name:USB-HID Keyboard Mouse:
```
which I think is a bit nicer

Reply:
I'm not the person who can do that work.  I can't make sense of how these files interact; lots of things seem to be autogenerated based on inputs that aren't immediately discernable.  I was hoping to keep a light touch since this is a minor issue and there is precedent among similar devices.